libp2p/protocols/pubsub/pubsub.nim

```diff
@@ -266,6 +303,18 @@ proc sendHelper*(p: PubSub,
 
   return (published, failed)
 
+proc sendAndCleanup*(p: PubSub,
```

Comment:
the name kind mislead me.. it gave me the illusion that the peer would be cleaned up all the time , maybe no need to mention the cleanup part.. dunno just sharing my feeling 😄

Reply:
hmm maybe something like `publishAndDisconnect`, I wan't to make it explicit that this helper will cleanup failed peers/streams.